store/analysis.js

```diff
@@ -73,7 +74,8 @@ export const actions = {
             acf,
             categories,
             country,
-            user
+            user,
+            footnote: content.rendered.match(regex)
```

Comment:
```suggestion
            footnotes: content.rendered.match(regex)
```

I'd make this plural because you're expecting to get multiple matches.